src/styles/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid $white-color;
```

Comment:
According to the Code Style Rules, color values must be stored in variables. Please, define a variable for the color `#fff` and use it instead. This will make your code more maintainable and consistent.